scout/run.py

```diff
@@ -4738,12 +4739,23 @@ def main(opts: argparse.NameSpace):  # noqa: F821
     # that do not have a matching ECM definition, which will be excluded
     for mn in active_meas_all:
         if mn not in [m["name"] for m in meas_summary]:
-            print("WARNING: ECM '" + mn + "' in 'run_setup.json' active " +
-                  "list does not match any of the ECM names found in " +
-                  "./ecm_definitions JSONs and will not be simulated")
+            warnings.warn(
+                "WARNING: ECM '" + mn + "' in 'run_setup.json' active " +
+                "list does not match any of the ECM names found in " +
+                "./ecm_definitions JSONs and will not be simulated")
```

Comment:
I missed this when updating filepaths originally

```suggestion
                f"{fp.ECM_DEF} JSONs and will not be simulated")
```